pkg/management/postgres/webserver/backup_client.go

```diff
@@ -61,7 +72,11 @@ func (c *BackupClient) Status(ctx context.Context, podIP string) (*BackupResultD
 		return nil, err
 	}
 
-	return executeRequest[BackupResultData](ctx, c.cli, req)
+	res, err := executeRequestWithError[BackupResultData](ctx, c.cli, req, false)
```

Comment:
We evaluated the body errors in the old code as an error response code. This seems wrong.
The caller should determine if that error is relevant to the backup that is trying to run.